app/views/chargeback_rate/_cb_rate_show.html.haml

```diff
@@ -60,7 +60,7 @@
         %td
           = tier.start ? tier.start : 0
         %td
-          = tier.finish ? tier.finish : Float::INFINITY.to_s
+          = tier.finish ? _((tier.finish).to_s) : _(Float::INFINITY.to_s)
```

Comment:
Where is the content of `tier.finish` being collected for catalog inclusion? Same for `Float::INFINITY`.

Reply:
Not 100% sure what you're asking here.

Reply:
So `_((tier.finish).to_s)` as well as `_(Float::INFINITY.to_s)` are both dynamic applications of gettext, right? Meaning that the inside of `_()` in both cases is dynamic and can have (in theory) arbitrary value. So what I'm asking here is where are those potential values being collected for inclusion to the catalog? (so that the potential strings exist in translations).

Reply:
well from what I can tell tier.finish seems to always be some floating-point number. The reason I wrapped it in a gettext was so that it translates the number infinity whenever it comes up. So infinity should be the only string that would need to exist in translations (since infinity gets translated I guess it already exists).

Reply:
In that case the above line is not quite correct and in my opinion it shoud read:
```ruby
= tier.finish == Float::INFINITY ? _('Infinity') : tier.finish
```